server.js

```diff
@@ -0,0 +1,123 @@
+var Hapi = require('hapi');
+// var Path =
+var Inert = require('inert');
+var Vision = require('vision');
+var Handlebars = require('handlebars')
+
+var Request = require('request');
+
+var server = new Hapi.Server();
+
+server.connection({
+    port: 8000,
+    host: 'localhost'
+});
+
+server.register(Inert, function(err) {
+    if (err) {
+        throw err;
+    }
+
+    server.route({
+        method: "GET",
+        path: '/public/{path*}',
+        handler: {
+            directory: {
+                path: "./public",
+                listing: false,
+                index: false
+            }
+        }
+    });
+});
+
+server.register(Vision, function(err) {
+    if (err) {
+        throw err;
+    }
+
+    //configure template support
+    server.views({
+        engines: {
+            html: Handlebars
+        },
+        path: './public',
+        //need to figure out how to render partials and use default layout
+        // layoutPath: Path.join(__dirname, 'views/layouts')
+    })
+});
+
+server.route({
+    method: 'GET',
+    path: '/',
+    handler: function(request, reply) {
+        reply.view('index');
+    }
+});
```

Comment:
You can make it a bit simpler. You don’t need `vision` at all, `intert` is enough. Something like this is all you need

```js
server.register(Inert);

server.route({
    method: "GET",
    path: '/{path*}',
    handler: {
        directory: {
            path: "./public"
        }
    }
});
```

Note that it’s `path: '/{path*}',` not `path: '/public/{path*}',`. The `path` is the URL path, not the path on the file system, that is what you pass into the `handler.directory.path` option.

This route will now server all static assets from the `public/` folder at the root url. Makes sense?